packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -1106,18 +1110,19 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                 }
             }
 
+            const type = getGitType(summaryObject) as "blob" | "commit" | "tree" | "attachment";
```

Comment:
May want to put a note somewhere that these casts can be removed once prerelease of server is published.  Are you planning to do in a follow-up PR?

Reply:
Opened https://github.com/microsoft/FluidFramework/issues/4835 to follow up